src/oph/ehoks/palaute/opiskelija.clj

```diff
@@ -53,6 +53,9 @@
       (not (:osaamisen-hankkimisen-tarve hoks))
       "`osaamisen-hankkimisen-tarve` not set to `true` for given HOKS."
 
+      (and (= kysely :aloituskysely) (not (:sahkoposti hoks)))
+      "`sahkoposti` is mandatory for aloituskysely and it's currently missing."
```

Comment:
eiks niin että Paula just lisäs tän määrittelyyn siten että yhteystietojen puuttuminen ei estä palautteen muodostamista?

Reply:
Joo, näin ymmärsin. En oo korjannut koodia tuon jälkeen.